internal/manifest/list.go

```diff
@@ -65,6 +69,32 @@ type ListUpdate struct {
 	MediaType string
 }
 
+type ListOp int
+
+const (
+	listOpInvalid ListOp = iota
+	listOpAdd
+	listOpUpdate
```

Comment:
Uh, these two (unlike `listOpInvalid`) must be public for the new functionality to be usable at all.